modules/templates/helper.go

```diff
@@ -238,6 +239,15 @@ func DotEscape(raw string) string {
 	return strings.ReplaceAll(raw, ".", "\u200d.\u200d")
 }
 
+func Iif(condition bool, vals ...any) any {
```

Comment:
What is a "Iif"? Any better name possible?

Reply:
I think it is a proper name, and many languages/frameworks also use such name:

![image](https://github.com/go-gitea/gitea/assets/2114189/7aa6bcf6-fe04-4bf3-8957-ac615fbdf920)

![image](https://github.com/go-gitea/gitea/assets/2114189/ef7cacf8-28a2-4d4d-b1d3-d2f55911b776)

Reply:
I see, that is certainly useful for nested template expressions, e.g. when you want to do `{{foo (if cond trueVal falseVal)}}`. I'm actually surprised this is no part of go template.